src/components/variant/Variant.tsx

```diff
@@ -12,6 +13,7 @@ interface State {
     "status": string,
     "count": number
   }>,
+  redirectToBuild: boolean
```

Comment:
Let's name this something that describes the state rather than what it does

Reply:
Fixed